### DISCUSSION TITLE
Problem Establishing internet connectivity for OAI RAN (gNB) and OAI UE using two USRPs

### DISCUSSION THREAD

**Discussion by rajyay** [magma/magma#15452] [General]
Hi Community, Can someone help me to establish the internet connectivity for setup using the setup consisting of
Openairinterface RAN gNB in One machine with USRP X310,
Openairinterface UE in other machine with USRP B210 and
Magma 5G Core in different machine
![image](https://github.com/user-attachments/assets/2a03c736-d22f-4b13-ac7f-bffa36c502f3)

[show-tech  report.zip](https://github.com/user-attachments/files/16257412/show-tech.report.zip)

**Comment by javieraubert** [comment]
The commands for the tcpdump failed, you can see the files are empty and that on the log it said command timeout, can you double check that, but more important, can you provide captures of both interfaces while trying to access the internet from an UE?

**Reply by javieraubert** [reply]
Also this are empty:
Command: service magma@* status
Command: service sctpd status
Command: show_gateway_info.py
Command: checkin_cli.py
Command: mobility_cli.py get_subscriber_table
Command: pipelined_cli.py debug display_flows
Command: enodebd_cli.py get_all_status

Please provide those information.

**Reply by rajyay** [reply]
please find the eth0 and eth1 capture files and docker status logs as you have asked
[capture eth0 eth1.zip](https://github.com/user-attachments/files/16271121/capture.eth0.eth1.zip)
[docker status.docx](https://github.com/user-attachments/files/16271122/docker.status.docx)

**Comment by rajyay** [comment]
Hi, please find the attached files if it helps you in debugging my issue now. For your information i have used USRP B210 as gNB this time instead of USRP X310 all others remaining same. gnbcapture.pcap is the captured packets in the interface of gNB connected to core
[capture etho and eth1.zip](https://github.com/user-attachments/files/16336094/capture.etho.and.eth1.zip)

[gnblog.docx](https://github.com/user-attachments/files/16335906/gnblog.docx)

**Reply by rajyay** [reply]
gnb log output and ip route output when UE is pinging to 8.8.8.8 is shown in the attached image 
![image](https://github.com/user-attachments/assets/71347b95-1e0c-4b38-9a73-238ba2dce973)
![image](https://github.com/user-attachments/assets/5efa0018-8090-4104-b13d-062b7ffd68f8)

**Reply by jblakley** [reply]
The gNB pcap shows successful PDU session establishment and the ping attempts are coming in through the GTP Tunnel to the AGW. That implies the issue is either somewhere in the AGW or in the routing to the internet. I don't see the ping attempt in the `eth0` pcap. That implies the AGW isn't forwarding the ping attempt. Can you run the ping test again while capturing from `any` and post that pcap?

Note: if your original `eth0` pcap didn't capture during the ping attempts, please rerun that.

**Comment by lucasgonze** [comment]
Please let us know if this is still open.

**Reply by rajyay** [reply]
yes, this is still open
